test_data/yam/agama_auto_sle.yaml

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
why is needed this file?

Reply:
I detected that it is required by agama_auto to start installation.

Reply:
some mistake somewhere, please drop this file, anyway this workaround is only for interactive.

Reply:
could you drop completely this file, because we are not testing partitioning at the moment, so it makes no sense. we should have agama_sle.yaml I guess.

Reply:
b921578fbd0b3cdce810a39af267440600653053